src/KeyboardInputSchemes.js

```diff
@@ -108,43 +108,58 @@ const ExtendedKeyboardSequences: KeyboardInputScheme = {
             },
             actions: {
                 keyDef: { code: "KeyA", key: "a" },
-                actionName: "focusActions"
+                actionName: "focusActions",
+                description: "Move focus to the actions panel"
             },
             appHeader: {
                 keyDef: { code: "KeyH", key: "h" },
-                actionName: "focusAppHeader"
+                actionName: "focusAppHeader",
+                description: "Move focus to the page title"
             },
             addNodeToggle: {
                 keyDef: { code: "KeyT", key: "t" },
-                actionName: "focusAddNodeToggle"
+                actionName: "focusAddNodeToggle",
+                description: "Move focus to the add node toggle",
+                icon: "A23.png",
```

Comment:
I would suggest naming all of these a little more specifically.  For example, we'll want to remove the left180 and right180 definitions once those are no longer focusable in the UI.  It should be at least vaguely clear to someone doing that or similar work what each image represents.  Just a little bit of context is fine, like `focus-add-node-toggle.png` here.